proto/storage/report_configuration.proto

```diff
@@ -69,8 +70,11 @@ message VulnerabilityReportFilters {
     google.protobuf.Timestamp since_start_date = 7;
   }
   repeated SimpleAccessScope.Rules access_scope_rules = 8;
-  bool include_nvd_cvss = 9;
-  bool include_epss_probability = 10;
+}
+
+message VulnerabilityReportOptionalColumns {
```

Comment:
nit:  not a huge fan of the name ending in `Columns`.

Reply:
This says VulnReportColumns so  it is clear this is referring to csv columns